src/tile-text/tile-text.scss

```diff
@@ -83,7 +83,7 @@ $grey: #e1e0e4;
     }
 
     .tile-title {
-      color: $mid-grey;
+      color: #4a4a4a;
```

Comment:
yeah it should be but the toolkit isn't in tiles and since tiles is being completely sacked off soon, I thought just for the sake of a font colour change i'd use the hex value rather than pull in the whole toolkit :)